src/bindings/python/src/openvino/runtime/__init__.py

```diff
@@ -82,3 +82,10 @@
 Node.__le__ = opset10.less_equal
 Node.__gt__ = opset10.greater
 Node.__ge__ = opset10.greater_equal
+
+
+# Import MO Python API
+try:
+    from openvino.tools.mo import convert_model
+except ImportError:
+    pass
```

Comment:
Please provide a stub implementation that shows help message for the user about openvino-dev.